core/src/main/kotlin/com/alecstrong/sql/psi/core/postgresql/PostgreSql.bnf

```diff
@@ -93,6 +94,11 @@ bind_parameter ::= ( '?' | ':' {identifier} ) {
   override = true
 }
 
+// TODO include the sequence creation clause '(' create_sequence_stmt ')'
```

Comment:
This should probably be opened as another issue, as we currently don't have support for _any_ sequence creation.

Reply:
remove the TODO and create the issue?

Reply:
For sure.

Reply:
#296